### DISCUSSION TITLE
Searching functionalities of the site build on Ananke theme

### DISCUSSION THREAD

**Discussion by fewsteps** [gohugo-ananke/ananke#641] [2 - Q&A / Support]
The theme is awesome but i was looking for whether it has any search functionality of the site? 

If anyone is able to implement that on this theme, can you share your knowledge to implement that feature?

**Comment by RoninTech** [comment]
I have just integrated the [pagefind](https://pagefind.app/) search tool on my Ananke based Hugo site.

Details are [here](https://discourse.gohugo.io/t/hugo-pagefind-questions/45222).  Very easy to get working and very powerful.  See it in action [here](https://carryonrtw.com/search/).
